### PR TITLE
fix  dubbo-samples-generic it 

### PR DESCRIPTION
**Pull request by shengchaojie**
fix  dubbo-samples-generic it

### CODE REVIEW

File: dubbo-samples-generic/dubbo-samples-generic-impl/case-configuration.yml [hunk 1]
```diff
@@ -18,6 +18,7 @@ from: app-builtin-zookeeper.yml
 
 props:
   project_name: dubbo-samples-generic-impl-provider
```
Comment (kylixs) [inline dubbo-samples-generic/dubbo-samples-generic-impl/case-configuration.yml:20]: project_name  建议改成目录名吧

Comment (shengchaojie) [reply dubbo-samples-generic/dubbo-samples-generic-impl/case-configuration.yml:20]: 👌

Comment (kylixs) [reply dubbo-samples-generic/dubbo-samples-generic-impl/case-configuration.yml:20]: 请将project_name 改为dubbo-samples-generic-impl

Comment (shengchaojie) [reply dubbo-samples-generic/dubbo-samples-generic-impl/case-configuration.yml:20]: 好滴，马上提

**Review by kylixs — APPROVED** [review]
Thanks for contribution!

**Pull request merged**